controllers/cloud.redhat.com/providers/featureflags/localfeatureflags.go

```diff
@@ -229,7 +229,7 @@ func (ff *localFeatureFlagsProvider) Provide(_ *crd.ClowdApp) error {
 	return nil
 }
 
-func makeLocalFeatureFlags(o obj.ClowdObject, objMap providers.ObjectMap, _ bool, nodePort bool) error {
+func makeLocalFeatureFlags(_ *crd.ClowdEnvironment, o obj.ClowdObject, objMap providers.ObjectMap, _ bool, nodePort bool) error {
```

Comment:
Why did we add an unused param for ClowdEnvironment? I don't understand.

Reply:
This is to enable it to be used in one particular provider only (redis)

Reply:
And to satisfy the interface ;)